### DISCUSSION TITLE
Importing the INI file created when Exporting Layers

### DISCUSSION THREAD

**Discussion by JacobCarrell** [sn4k3/UVtools#155] [Q&A] [answered]
How do I use the file created when you export layers? I'm trying to do some mostly programmatic file creation (I haven't found any good CBDDLP v3 libraries), and noticed that an INI with the per layer information is created when you export all layers, but I do not see a way to import this information. Can somebody please tell me how to use that file? If it hasn't been implemented, I'll give it a go. A nudge in the right direction would be appreciated in that case.

Thanks!

**Comment by sn4k3** [comment, marked as answer]
UVtools create those files for expose values to user, it's just informative.
Looks like you want to create a new file from other? You can use UVtools core in C# or use a simple PowerShell script.
More information here: https://github.com/sn4k3/UVtools/tree/master/UVtools.Scripts

**Reply by JacobCarrell** [reply]
That'll work. Thanks!
